src/Mod/Arch/ArchCommands.py

```diff
@@ -1110,10 +1103,7 @@ def exportCSV(self):
         if rows:
             filename = QtGui.QFileDialog.getSaveFileName(QtGui.QApplication.activeWindow(), translate("Arch","Export CSV File"), None, "CSV file (*.csv)");
             if filename:
-                if sys.version_info.major < 3:
-                    mode = 'wb'
-                else:
-                    mode = 'w'
+                mode = 'w'
                 with open(filename[0].encode("utf8"), mode) as csvfile:
```

Comment:
```suggestion
                with open(filename[0].encode("utf8"), "w") as csvfile:
```

Reply:
Thanks for your review!

your changes are further beyond what the PR does. I'll merge it now. Please make a new PR for further code simplification

Reply:
I don't agree. The only reason for the mode variable was changing between `w` and `wb` between Python 2 and 3. Removing the code changing those should also cause the removal of the variable.

My other comments were about removal of code explaining the code you just deleted. If you remove some code, you should also remove the comments that belonged to it.

Reply:
> I don't agree. The only reason for the mode variable was changing between w and wb between Python 2 and 3. 

Hmm, OK. I committed this change now.

> My other comments were about removal of code explaining the code you just deleted. If you remove some code, you should also remove the comments that belonged to it. How does that not belong to this PR

I don't agree hereby. why is range copied to xrange? There must be a comment for this why this is done since this is not obvious.
When there should not be a comment, the whole xrange variable should got. I made a PR for this, please review:
https://github.com/FreeCAD/FreeCAD/pull/7309

Reply:
> I don't agree hereby. why is range copied to xrange? There must be a comment for this why this is done since this is not obvious.
True